devices/patches/usb_otg/fs_l4+_global.yaml

```diff
@@ -0,0 +1,272 @@
+GOTGCTL:
```

Comment:
According to rm0432, some of these fields should be read only. Should that be specified here?